notebooks/analyze_model_output.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    yaml_file = os.path.join(folder, dataset_name, 'data_info.yaml')
</pre><p>Is data_info.yaml the right name or config.yaml?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>